website/docs/components/pagination/partials/code/component-api.md

```diff
@@ -161,6 +164,9 @@ Used to indicate which of the provided `sizes` options is pre-selected. Normally
 <C.Property @name="onChange" @type="function">
 Callback function invoked (if provided) when the page size selector is changed. The function receives the value of the "page size" as argument (an integer number).
 </C.Property>
+<C.Property @name="label" @type="string" @values={{array "Items per page" "Custom string" }} @default="Items per page">
```

Comment:
Nitpick: we tend to have the events (eg. `onSomething`) after the other "normal" arguments. Can you please move the `label` before the `onChange` here? Thanks